src/components/Course/Main.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="main">
+    <el-row style="margin: 4% 0 10px 0">
+      <div class="tle">
+        <div style="margin-right: 5%;">
+          <span style="font-size: 30px;" >{{ coinfo.name }}</span>
+        </div>
+        <div class="blackline" style="margin-right: 5%"></div>
+        <div style="margin-top: 5px">
+          <span style="font-size: 20px;font-style: italic;">{{ coinfo.semester }}</span>
+        </div>
+      </div>
+    </el-row>
+    <el-row>
+      <el-table
+        :data="costate"
+        style="width: 100%"
+        stripe>
+        <el-table-column
+          prop="name"
+          label="NAME"
+          >
+        </el-table-column>
+        <el-table-column
+          prop="state"
+          label="STATUS"
+          width="180">
+        </el-table-column>
+        <el-table-column
+          prop="release"
+          label="RELEASE">
+        </el-table-column>
+        <el-table-column
+          prop="due"
+          label="DUE(CST"
+          width="180">
+        </el-table-column>
+      </el-table>
+    </el-row>
+  </div>
+</template>
+<script>
+export default {
+  data () {
+    return {
+      costate: [{
+        name: '',
+        state: '',
+        release: '',
+        due: '',
+        check: true
+      }],
+      coinfo: {
+        name: '',
+        group: '',
+        instructors: ['']
+      },
+      tableData2: [{
```

Comment:
useless

Reply:
Is there no need to initialize the list?

Reply:
tableData is not used in the code